attalos/imgtxt_algorithms/main.ipynb

```diff
@@ -0,0 +1,168 @@
+{
```

Comment:
@ymt123 wants to remove this notebook (`main.ipynb`) from this PR, so I will defer to him on this.